pkg/blob/nodeserver.go

```diff
@@ -213,15 +224,28 @@ func (d *Driver) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeRe
 
 	klog.V(2).Infof("target %v\nprotocol %v\n\nvolumeId %v\ncontext %v\nmountflags %v\nmountOptions %v\nargs %v\nserverAddress %v",
 		targetPath, protocol, volumeID, attrib, mountFlags, mountOptions, args, serverAddress)
-	cmd := exec.Command("blobfuse", strings.Split(args, " ")...)
 
-	cmd.Env = append(os.Environ(), "AZURE_STORAGE_ACCOUNT="+accountName)
-	cmd.Env = append(cmd.Env, "AZURE_STORAGE_BLOB_ENDPOINT="+serverAddress)
-	cmd.Env = append(cmd.Env, authEnv...)
+	blobfuseProxyEndpoint := "unix:///var/lib/kubelet/plugins/blobfuseproxy.sock"
```

Comment:
should define an endpoint here:
https://github.com/kubernetes-sigs/blob-csi-driver/blob/73aa89661f450327963e2e68f1b1155b73503538/pkg/blobplugin/main.go#L39

and set here: 
https://github.com/kubernetes-sigs/blob-csi-driver/blob/73aa89661f450327963e2e68f1b1155b73503538/deploy/csi-blob-node.yaml#L83-L85